SEARCH.md

```diff
@@ -142,12 +142,11 @@ The aliases are used at index-creation time, so if this file changes then the in
 
 It seems like Netlify uses the deployed functions rather than the ones in a branch when building a preview, so we need to take care when testing these. A good approach is to use the [Netlify CLI](https://www.netlify.com/products/cli/) locally. For example, to test the search function:
```

Comment:
This might be too much clarity but let me try:
* Our codebase includes some functions as well as the website
* Netlify deploys a preview of website changes, and has working functions BUT these seem to be (we don't seem very certain) the live version of the functions, not the branch version
* As a result, when you want to make and test changes to the functions, you must take care.
* An alternative is to run things locally, where both the codebase and function versions from the current branch will be used.

Also to note - this information is very specific to our current hosting platform, Netlify, which is being changed in the coming weeks.